source/mir/ndslice/slice.d

```diff
@@ -3730,3 +3736,39 @@ version(mir_test) unittest
     assert(immdf.label!0[0] == 1);
     assert(immdf.label!1[1] == 2);
 }
+
+version(mir_test) unittest
```

Comment:
mark all of them as @safe pure nothrow

Reply:
and group into a single unittest

Reply:
cast isn't allowed in @safe. I'll mark them pure nothrow

Reply:
Addressed in b780bee